src/main/webapp/gast/person.jsp

```diff
@@ -1,11 +1,38 @@
+<%@page import="java.util.Set"%>
+<%@page import="de.uni_tuebingen.ub.nppm.model.Einzelbeleg"%>
+<%@page import="de.uni_tuebingen.ub.nppm.model.Person"%>
 <%@ page import="de.uni_tuebingen.ub.nppm.util.Language" isThreadSafe="false" %>
+<%@ page import="de.uni_tuebingen.ub.nppm.exception.*" isThreadSafe="false" %>
 <%@ include file="../configuration.jsp"%>
 <%@ include file="../functions.jsp" %>
 
 <jsp:include page="../dofilter.jsp" />
 
-<%
-   int id = Integer.parseInt(request.getParameter("ID"));
+<%    int id = Integer.parseInt(request.getParameter("ID"));
+
+    if (PersonDB.getById(id) == null) {
+        throw new IdNotFoundException("Person ID P" + String.valueOf(id) + " ist nicht vorhanden");
+    } else {
+        Person person = PersonDB.getById(id);
```

Comment:
Hier haben wir die Person-Abfrage doppelt, in Zeile 13 und in 16 (lieber schon z.B. in Zeile 12 zuweisen und in 13 die Variable gegen null vergleichen)

Reply:
(ist wohl auch bei anderen so, z.B. namenkommentar, mghlemma, ...)